src/guide/introduction.md

```diff
@@ -239,6 +239,20 @@ const app = Vue.createApp({
 app.mount(...)
 ```
 
+Another way to register a component is to use the `component()` method on the `app` object, passing in the name of the component as the first attribute and an options object as the second:
```

Comment:
**suggestion**: I agree that `app.component` looks a bit weird here but this explanation rather belongs to `Component In-Depth` section. What if we refactor the components below to be local components instead of globally registered ones?

Reply:
I do think that's a better idea, since it would make it less confusing for learners while also reducing redundant documentation. I'll make these changes. Thanks!

Reply:
Thank you for the super-fast fixes applied! Happy to merge them to the docs 🎉